### PR TITLE
Update deprecated material-ui method

### PR DESCRIPTION
**Pull request by dasanra**
There is a constant warning in console for `safe-react` warning about a deprecated method in material-ui

We are also updating it here in order to minimize the warnings

### CODE REVIEW

**Review by mikheevm — APPROVED** [review]
Danke!

File: src/utils/CopyToClipboardBtn/index.tsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ const CopyToClipboardBtn = ({
   };
 
   const onButtonBlur = (): number =>
-    setTimeout((): void => setClicked(false), 300);
+    window.setTimeout((): void => setClicked(false), 300);
```
Comment (mikheevm) [inline src/utils/CopyToClipboardBtn/index.tsx:57]: why was this needed? if someone would use it in SSR env it will fail. If we need to use the window it's better to check with `if (typeof window !== 'undefined)`

Comment (dasanra) [reply src/utils/CopyToClipboardBtn/index.tsx]: There is a typescript error in that line when calling setTimeout directly we can leave as it was until now

Comment (dasanra) [reply src/utils/CopyToClipboardBtn/index.tsx]: Now, it seems we have to take some action there
https://github.com/gnosis/safe-react-components/pull/146/checks?check_run_id=3359298170#step:4:70

Comment (mikheevm) [reply src/utils/CopyToClipboardBtn/index.tsx]: you can also not return it by wrapping the function body with curly braces and not including return statement
```
const onButtonBlur = (): void => {
  setTimeout((): void => setClicked(false), 300)
  }
